examples/aides_logement/code_construction_reglementaire.catala_fr

```diff
@@ -4645,10 +4646,10 @@ constatée.
 
 ```catala
 champ d'application CalculAllocationLogementAccessionPropriété:
-  définition abattement_dépense_nette_minimale de allocation_mensuelle
+  définition abattement_dépense_nette_minimale de aide_calculée_1_8_d832_24
```

Comment:
That's a weird name no ?

Reply:
Why didn't you keep `allocation_mensuelle` everywhere ?

Reply:
Hum I had to choose between the two, and kept the one that felt more precise, but I can switch.

Reply:
Yeah in this case `allocation_mensuelle` is preferable.